13-transport.Rmd

```diff
@@ -326,86 +329,106 @@ There are three main ways to compute routes from OD data:
+Most routing engines return route level by default; multi-modal engines can provide outputs at the 'leg' level (one feature per continuous movement by a single mode of transport) while the **cyclestreets** package, for example, provides segment level data by default.
+When working with segment or leg-level data, route-level statistics can be returned by grouping by columns representing trip start and end points and summarizing/aggregating columns containing segment-level data.
+
+Instead of routing\index{routing} *all* desire lines generated in the previous section, we will focus on the desire lines\index{desire lines} of policy interest.
+Doing routing on a subset of the data before trying to compute routes for a large number of OD pairs is often sensible: routing can be time and memory-consuming: route datasets are usually substantially larger than desire line datasets because they have more detailed geometries and often have more attributes such as road name.
+
+We will filter-out a subset of the desire lines with a focus on estimating cycling potential based on the observation that the benefits cycling trips are greatest when they replace car trips and that relatively short trips are more likely to be cycled than long trips [@lovelace_propensity_2017].
 Clearly, not all car trips can realistically be replaced by cycling.
-However, 5 km Euclidean distance (or around 6-8 km of route distance) can realistically be cycled by many people, especially if they are riding an electric bicycle ('ebike').
-We will therefore only route desire lines\index{desire lines} along which a high (300+) number of car trips take place that are up to 5 km in distance.
-This routing\index{routing} is done in the code chunk below by the **stplanr**\index{stplanr (package)} function `route()`, which creates `sf` objects representing routes on the transport network\index{network}, one for each desire line.
+Trips between 2.5 and 5 km Euclidean distance (or around 3 km, below which trips are can be made by walking or scooter, to 6-8 km of route distance) have a relatively high probability of being cycled, and the maximum distance increases when trips are made by [electric bike](https://www.sciencedirect.com/science/article/pii/S0967070X21003401).
+<!-- Todo: add reference (RL 2022-07) -->
+Considering this, and the utility of a small route dataset for testing, we will compute routes for the subset of desire lines\index{desire lines} along which a high (100+) number of car trips take place that are 2.5 to 5 km in length:
 
 ```{r 13-transport-17, message=FALSE}
-desire_lines$distance = as.numeric(st_length(desire_lines))
-desire_carshort = dplyr::filter(desire_lines, car_driver > 300 & distance < 5000)
+desire_lines$distance_km = as.numeric(st_length(desire_lines)) / 1000
+desire_lines_short = desire_lines %>% 
+  filter(car_driver >= 100 & distance_km <= 5 & distance_km >= 2.5)
```

Comment:
These slightly weird subsetting criteria were selected to make the results look good FYI: different values I tried led to too many random lines far from the city centre.